src/DynamoCoreWpf/ViewModels/PackageManager/PackageManagerClientViewModel.cs

```diff
@@ -498,18 +498,19 @@ internal void ExecutePackageDownload(string name, PackageVersion package, string
             var pmExt = DynamoViewModel.Model.GetPackageManagerExtension();
             if (result == MessageBoxResult.OK)
             {
+                package.full_dependency_ids.Count == package.full_dependency_versions.Count
```

Comment:
I wonder what the best assert utility would be to check that these 2 arrays are in sync...
@aparajit-pratap any ideas ?

Reply:
`Debug.Assert`?